app/helpers/rails_admin/application_helper.rb

```diff
@@ -28,7 +28,7 @@ def edit_user_link
         content_tag(:span, _current_user.email),
       ].compact.join.html_safe
       if (edit_action = RailsAdmin::Config::Actions.find(:edit, controller: controller, abstract_model: abstract_model, object: _current_user)).try(:authorized?)
-        link_to content, rails_admin.url_for(action: edit_action.action_name, model_name: abstract_model.to_param, id: _current_user.id, controller: 'rails_admin/main')
+        link_to content, rails_admin.url_for(action: edit_action.action_name, model_name: abstract_model.to_param, id: _current_user.id, controller: 'rails_admin/main'), class: 'nav-link'
```

Comment:
Most of these kind of changes are to keep up with the change in Bootstrap 5, like `navbar` needing `nav-item` and `nav-link` class for elements inside it.
https://getbootstrap.com/docs/5.1/components/navbar/